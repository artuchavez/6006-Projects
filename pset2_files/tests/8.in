min-heap
Yoyo,Ma
3,2
Yoyo,Ma
1,1
