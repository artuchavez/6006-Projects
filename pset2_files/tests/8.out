Yoyo,Ma
1,1